openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -17,15 +17,15 @@ <h4 class="step__title">
                     {% if submission_start %}
                     <span class="step__deadline">
                         {# Translators: This string displays a date to the user, then tells them the time until that date.  Example: "available August 13th, 2014 (in 5 days and 45 minutes)" #}
-                        {% blocktrans with start_date=submission_start|timezone:"UTC"|date:"c" time_until=submission_start|timeuntil %}
-                        <span class="date ora-datetime" data-datetime="{{ start_date }}" data-string="available {date} (in {{ time_until }})" data-timezone="{{ user_timezone }}" data-language="{{ user_language }}"></span>
+                        {% blocktrans with start_date=submission_start|timezone:"UTC"|date:"d/M/y H:m" time_until=submission_start|timeuntil %}
```

Comment:
@diegomillan instead of hardcoding the format of the date can we use a localized date format or a feature flag?